cmd/vela-server/schedule.go

```diff
@@ -203,3 +212,20 @@ func processSchedule(ctx context.Context, s *library.Schedule, compiler compiler
 
 	return nil
 }
+
+func logError(ctx context.Context, database database.Interface, err error, schedule *api.Schedule) {
```

Comment:
the name of this function seems vague compared to what it actually does (log _and_ update the db record). maybe `handleError` would fit better? or move that db call outside of this function?

Reply:
Good suggestion, `handleError` is a more appropriate name. I'll use that.